### ISSUE TITLE
Swipable Tabs

### ISSUE THREAD

**Issue by deadcoder0904** [RCiesielczuk/HackerBuzz-ReactNative#2]
Currently swiping from the side doesn't work.

Like if I am on `New` tab & swipe left then it should take me to `Top` tab but it does not work.

**Comment by a-dilettante** [comment]
The main reason why it's disabled, it's because if you swipe from the left-hand-side of an item, then it'll try and favorite and that would conflict with swipe-able tabs. 

It's [this](https://github.com/RCiesielczuk/HackerBuzz-ReactNative/blob/master/js/containers/main/StoriesContainer.js#L34) prop that's causing the swiping to be disabled

**Comment by deadcoder0904** [comment]
Yep, that's true but I think that looks weird kinda. If you want to favorite you can double-tap & enabling Tab Views would be much better I guess. This might be much better DX.

& its kinda trouble to move your finger all the way above to click tabs rather than swipe. Bcz if I want to use this app, I want it to be less time-consuming & much faster.
